axum-extra/src/extract/cookie.rs

```diff
@@ -15,6 +15,9 @@ use http::{
 };
 use std::{convert::Infallible, fmt, marker::PhantomData};
 
+mod private;
+
+pub use self::private::PrivateCookieJar;
```

Comment:
Gonna move the plaintext and signed versions to their own mods in a follow up PR